### PR TITLE
Changes after review

### PR DESCRIPTION
**Pull request by MikeDodaro**

### CODE REVIEW

File: docs/azure-simple/README.md [hunk 1]
```diff
@@ -688,37 +688,6 @@ module.aks-gitops.module.flux.null_resource.deploy_flux: Creation complete after
 
 Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
 ```
-
```
Comment (jmspring) [inline docs/azure-simple/README.md]: Did you mean to take this section out?

Comment (MikeDodaro) [reply docs/azure-simple/README.md]: Yes, it is all in the following section.  Hemavathy found the double.

Comment (MikeDodaro) [reply docs/azure-simple/README.md]: In the previous section actually.

File: docs/azure-simple/README.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ Define the remainding fields:
 - `dns_prefix`: `testazuresimple`
 - `vnet_name`: `testazuresimplevnet`
 
-The, `gitops_ssh_key` is a *path* to the RSA private key we created under [Create an RSA Key for Logging Into AKS Nodes](#create-an-rsa-key-for-logging-into-aks-nodes)
+The, `gitops_ssh_key` is a *path* to the RSA private key we created under [Set Up Flux Manifest Repository](##set-up-flux-manifest-repository)
```
Comment (jmspring) [inline docs/azure-simple/README.md]: Use one # not two

Comment (MikeDodaro) [reply docs/azure-simple/README.md]: Thanks, fixed.

**Pull request merged**